samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -56,6 +56,8 @@ import org.apache.samza.task._
 import org.apache.samza.util.Util
 import org.apache.samza.util._
 import org.apache.samza.{SamzaContainerStatus, SamzaException}
+import org.apache.samza.diagnostics.DiagnosticsAppender;
+import org.apache.log4j.Logger
```

Comment:
We are not using log4j directly in samza-core. We always use slf4j libraries in all our source code, except for the samza-log4j module.